### DISCUSSION TITLE
can this be used for gaming?

### DISCUSSION THREAD

**Discussion by DOOMMARINE117** [siderolabs/talos#7376] [General]
i was looking at distros in the meantime awhile haiku is still being devolved.

i saw this. 

can this linux distro of talos can be used for gaming?

**Comment by steverfrancis** [comment]
Depends what you mean... Talos Linux is well suited to deploy Kubernetes to host gaming services (and in fact is in use by some very large gaming providers).
It cannot be used to play games on, lacking the ability to log in to it, or support for graphics, or users, or many other things. It's not designed for that.
